cs-jclouds/src/main/java/io/cloudslang/content/jclouds/actions/instances/ModifyInstanceAttributeAction.java

```diff
@@ -0,0 +1,217 @@
+package io.cloudslang.content.jclouds.actions.instances;
```

Comment:
since, as a whole, the jclouds package is going to be transitioned to cs-amazon, I suggest we update the package instances accordingly